src/Validation/Validation.php

```diff
@@ -181,25 +187,27 @@ public static function blank($check)
             'Validation::blank() is deprecated.'
         );
 
-        return !static::_check($check, '/[^\\s]/');
+        return ! static::_check($check, '/[^\\s]/');
     }
 
     /**
      * Validation of credit card numbers.
      * Returns true if $check is in the proper credit card format.
      *
-     * @param string $check credit card number to validate
-     * @param string|array $type 'all' may be passed as a string, defaults to fast which checks format of most major credit cards
-     *    if an array is used only the values of the array are checked.
-     *    Example: ['amex', 'bankcard', 'maestro']
-     * @param bool $deep set to true this will check the Luhn algorithm of the credit card.
-     * @param string|null $regex A custom regex can also be passed, this will be used instead of the defined regex values
+     * @param string       $check credit card number to validate
+     * @param string|array $type  'all' may be passed as a string, defaults to fast which checks format of most major
+     *                            credit cards if an array is used only the values of the array are checked. Example:
+     *                            ['amex', 'bankcard', 'maestro']
+     * @param bool         $deep  set to true this will check the Luhn algorithm of the credit card.
+     * @param string|null  $regex A custom regex can also be passed, this will be used instead of the defined regex
+     *                            values
+     *
      * @return bool Success
      * @see \Cake\Validation\Validation::luhn()
      */
     public static function cc($check, $type = 'fast', $deep = false, $regex = null)
     {
-        if (!is_scalar($check)) {
+        if (! is_scalar($check)) {
```

Comment:
Why are you changing this to an invalid CS? There must be no space after casts.
Also, please only change exactly the lines you want to change, no other noise changes.

Reply:
Sure. Sorry, my bad